### ISSUE TITLE
Clean up orog fix directory

### ISSUE THREAD

**Issue by WalterKolczynski-NOAA** [NOAA-EMC/global-workflow#1206]
**Description**
The non-fractional grid files are no longer needed in the `fix/orog` directory. Additionally, there is some clean-up of file names to be had (removing the resolution, using names used at runtime, etc.).

Also, the top of `fix/orog` has some files that look like they are scripts or source files for generating the orog files that can be removed from fix and placed elsewhere. Will seek guidance from @yangfanglin on these.

**Dependencies**
Depends on #863 / #1182 to update the fix version in workflow, but the new fix version can be created and staged in the interim.

**Comment by WalterKolczynski-NOAA** [comment]
The mosaic and grid tile filenames are built-in to the grid_spec file, so those filenames will remain unchanged for now.

There is a preview available on Orion at `/work2/noaa/global/wkolczyn/noscrub/fix_work/orog/20220805` pending resolution of the top-level stuff. (The date will be updated when we are ready to copy to authoritative fix.)

Example listing:
```
Orion-login-2 C384.mx025/>pwd
/work2/noaa/global/wkolczyn/noscrub/fix_work/orog/20220805/C384.mx025
Orion-login-2 C384.mx025/>ls -R
.:
C384_grid.tile1.nc  C384_grid.tile6.nc                                              oro_data.tile1.nc  oro_data.tile6.nc
C384_grid.tile2.nc  C384_mosaic.nc                                                  oro_data.tile2.nc  remap_weights_C384_0p125deg.nc
C384_grid.tile3.nc  fix_sfc                                                         oro_data.tile3.nc  remap_weights_C384_0p25deg.nc
C384_grid.tile4.nc  fv3_SCRIP_C384_GRIDSPEC_lon1536_lat768.gaussian.bilinear.nc     oro_data.tile4.nc  remap_weights_C384_0p5deg.nc
C384_grid.tile5.nc  fv3_SCRIP_C384_GRIDSPEC_lon1536_lat768.gaussian.neareststod.nc  oro_data.tile5.nc  remap_weights_C384_1deg.nc

./fix_sfc:
facsf.tile1.nc                slope_type.tile1.nc       soil_type.tile1.nc              vegetation_greenness.tile1.nc
facsf.tile2.nc                slope_type.tile2.nc       soil_type.tile2.nc              vegetation_greenness.tile2.nc
facsf.tile3.nc                slope_type.tile3.nc       soil_type.tile3.nc              vegetation_greenness.tile3.nc
facsf.tile4.nc                slope_type.tile4.nc       soil_type.tile4.nc              vegetation_greenness.tile4.nc
facsf.tile5.nc                slope_type.tile5.nc       soil_type.tile5.nc              vegetation_greenness.tile5.nc
facsf.tile6.nc                slope_type.tile6.nc       soil_type.tile6.nc              vegetation_greenness.tile6.nc
maximum_snow_albedo.tile1.nc  snowfree_albedo.tile1.nc  substrate_temperature.tile1.nc  vegetation_type.tile1.nc
maximum_snow_albedo.tile2.nc  snowfree_albedo.tile2.nc  substrate_temperature.tile2.nc  vegetation_type.tile2.nc
maximum_snow_albedo.tile3.nc  snowfree_albedo.tile3.nc  substrate_temperature.tile3.nc  vegetation_type.tile3.nc
maximum_snow_albedo.tile4.nc  snowfree_albedo.tile4.nc  substrate_temperature.tile4.nc  vegetation_type.tile4.nc
maximum_snow_albedo.tile5.nc  snowfree_albedo.tile5.nc  substrate_temperature.tile5.nc  vegetation_type.tile5.nc
maximum_snow_albedo.tile6.nc  snowfree_albedo.tile6.nc  substrate_temperature.tile6.nc  vegetation_type.tile6.nc
```

**Comment by WalterKolczynski-NOAA** [comment]
Per Fanglin, @GeorgeGayno-NOAA should be the contact about the top-level orog apparent source data/scripts.

**Comment by GeorgeGayno-NOAA** [comment]
@WalterKolczynski-NOAA I assume you are asking about these files:

- convert.f90
- makefile
- run.lsf

**Comment by WalterKolczynski-NOAA** [comment]
@GeorgeGayno-NOAA Not just those, but also the other files at the base of the orog fix directory that mostly look like source files for the generation process:
```
clmgrb
clmgrb.index
convert.f90
geo_em.d01.lat-lon.2.5m.HGT_M.nc
GlobalLakeDepth.dat
GlobalLakeDepth_GLDBv3release.dat
GlobalLakeDepth_GLOBathy.dat
GlobalLakeDepth.txt
GlobalLakeStatus.dat
GlobalLakeStatus_GLDBv3release.dat
GlobalLakeStatus_MODISp.dat
GlobalLakeStatus.txt
GlobalLakeStatus_VIIRS.dat
gmted2010.30sec.flt
gmted2010.30sec.flt.ctl
gmted2010.30sec.int
gmted2010.30sec.int.ctl
gtopo30_gg.fine
gtopo30_gg.fine.nh
HGT.Beljaars_filtered.lat-lon.30s_res.nc
landcover30.fixed
makefile
run.lsf
thirty.second.antarctic.new.bin
TOP8M_avg.20I4.asc
TOP8M_max.20I4.asc
TOP8M_slm.80I1.asc
```

**Comment by GeorgeGayno-NOAA** [comment]
> @GeorgeGayno-NOAA Not just those, but also the other files at the base of the orog fix directory that mostly look like source files for the generation process:
> 
> ```
> clmgrb
> clmgrb.index
> convert.f90
> geo_em.d01.lat-lon.2.5m.HGT_M.nc
> GlobalLakeDepth.dat
> GlobalLakeDepth_GLDBv3release.dat
> GlobalLakeDepth_GLOBathy.dat
> GlobalLakeDepth.txt
> GlobalLakeStatus.dat
> GlobalLakeStatus_GLDBv3release.dat
> GlobalLakeStatus_MODISp.dat
> GlobalLakeStatus.txt
> GlobalLakeStatus_VIIRS.dat
> gmted2010.30sec.flt
> gmted2010.30sec.flt.ctl
> gmted2010.30sec.int
> gmted2010.30sec.int.ctl
> gtopo30_gg.fine
> gtopo30_gg.fine.nh
> HGT.Beljaars_filtered.lat-lon.30s_res.nc
> landcover30.fixed
> makefile
> run.lsf
> thirty.second.antarctic.new.bin
> TOP8M_avg.20I4.asc
> TOP8M_max.20I4.asc
> TOP8M_slm.80I1.asc
> ```

Some of these files are needed. And some are not used anymore. Let me investigate a bit.

**Comment by GeorgeGayno-NOAA** [comment]
UFS_UTILS uses GlobalLakeDepth.dat and GlobalLakeStatus.dat to add lakes to our model grids. So, keep those. The other "GlobalLake"files were added later and appear to be different versions of these files created from other sources. You will need to ask Shan Sun if they are needed.

**Comment by WalterKolczynski-NOAA** [comment]
@GeorgeGayno-NOAA I'm not saying get rid of them entirely, but do they belong in the general fix rather than some other place if they are only needed to generate the files actually used by the models (which _would_ go in fix)?

@shansun6 can you provide more information?

**Comment by ShanSunNOAA** [comment]
All files starting with GlobalLake are needed to generate fractional lake data in the orography file from different sources. The rest of the files on George's list are not from me or Ning Wang. Thanks for asking.

**Comment by WalterKolczynski-NOAA** [comment]
@shansun6 but they're not used by the model directly, right? Only to generate the grid data?

**Comment by ShanSunNOAA** [comment]
That is correct: all GlobalLake* files are used to generate oro_data.

**Comment by GeorgeGayno-NOAA** [comment]
> > @GeorgeGayno-NOAA Not just those, but also the other files at the base of the orog fix directory that mostly look like source files for the generation process:
> > ```
> > clmgrb
> > clmgrb.index
> > convert.f90
> > geo_em.d01.lat-lon.2.5m.HGT_M.nc
> > GlobalLakeDepth.dat
> > GlobalLakeDepth_GLDBv3release.dat
> > GlobalLakeDepth_GLOBathy.dat
> > GlobalLakeDepth.txt
> > GlobalLakeStatus.dat
> > GlobalLakeStatus_GLDBv3release.dat
> > GlobalLakeStatus_MODISp.dat
> > GlobalLakeStatus.txt
> > GlobalLakeStatus_VIIRS.dat
> > gmted2010.30sec.flt
> > gmted2010.30sec.flt.ctl
> > gmted2010.30sec.int
> > gmted2010.30sec.int.ctl
> > gtopo30_gg.fine
> > gtopo30_gg.fine.nh
> > HGT.Beljaars_filtered.lat-lon.30s_res.nc
> > landcover30.fixed
> > makefile
> > run.lsf
> > thirty.second.antarctic.new.bin
> > TOP8M_avg.20I4.asc
> > TOP8M_max.20I4.asc
> > TOP8M_slm.80I1.asc
> > ```
> 
> Some of these files are needed. And some are not used anymore. Let me investigate a bit.

These files are used as input to create model grids. So, if you want to move then to a different location, feel free.

**Comment by WalterKolczynski-NOAA** [comment]
Okay, here is my proposal of what should happen:

1. The code should be moved into a code repository with proper versioning (UFS_UTILS?)
2. The source data should be move out of fix and somewhere under control of whichever group is responsible for creating new grids
3. A README should be added describing how the data are being generated (e.g. "oro_data tile files are generated by regridding the USGS GMTED2010 30 arc-second (1/120 °) elevation data to the cube-sphere tile faces.") and contacts for each.

**Comment by yangfanglin** [comment]
Walter, these are all good ideas; however, I'd recommend we still keep those source files in the same place.  All the "fix" files are nevertheless saved on local platforms. They do not interfere with github.  By keeping them in the top layer of model resolution specific directories we always know what source files had been  used to create the files on the tiles.  I understand one can move them to a different location to make this directory cleaner, but there is always a risk of  losing track of these files during machine upgrades and staff changes.

**Comment by KateFriedman-NOAA** [comment]
Raw orog files used for orography fix file generation were moved to a top-level `raw/orog` folder and out from the `orog/20220805/` folder. The UFS_UTILS `develop` branch was updated to point to the needed raw files in the new location (https://github.com/ufs-community/UFS_UTILS/pull/758) and symlinks for specific files were created in `orog/20220805` to support older versions for now (per a request from @GeorgeGayno-NOAA):

```
-bash-4.2$ pwd
/scratch1/NCEPDEV/global/glopara/fix
-bash-4.2$ ll raw/orog/
total 31028328
-rw-r--r-- 1 glopara global  1866240000 May 31  2018 GlobalLakeDepth.dat
-rw-r--r-- 1 glopara global        2163 May 31  2018 GlobalLakeDepth.txt
-rw-r--r-- 1 glopara global  1866240000 Jul 11  2022 GlobalLakeDepth_GLDBv3release.dat
-rw-r--r-- 1 glopara global  1866240000 Jul 11  2022 GlobalLakeDepth_GLOBathy.dat
-rw-r--r-- 1 glopara global   933120000 May 31  2018 GlobalLakeStatus.dat
-rw-r--r-- 1 glopara global        2805 May 31  2018 GlobalLakeStatus.txt
-rw-r--r-- 1 glopara global   933120000 Jul 11  2022 GlobalLakeStatus_GLDBv3release.dat
-rw-r--r-- 1 glopara global   933120000 Jul 11  2022 GlobalLakeStatus_MODISp.dat
-rw-r--r-- 1 glopara global   933120000 Jul 11  2022 GlobalLakeStatus_VIIRS.dat
-rw-r--r-- 1 glopara global 11197441020 Jan 14  2021 HGT.Beljaars_filtered.lat-lon.30s_res.nc
-rw-r--r-- 1 glopara global    14762250 Mar 11  2017 TOP8M_avg.20I4.asc
-rw-r--r-- 1 glopara global    14762250 Mar 11  2017 TOP8M_max.20I4.asc
-rw-r--r-- 1 glopara global     3690563 Mar 11  2017 TOP8M_slm.80I1.asc
-rw-r--r-- 1 glopara global      616356 Mar 11  2017 clmgrb
-rw-r--r-- 1 glopara global        5538 Mar 11  2017 clmgrb.index
-rw-r--r-- 1 glopara global        6836 Aug 14  2017 convert.f90
-rw-r--r-- 1 glopara global   447900364 Feb  6  2021 geo_em.d01.lat-lon.2.5m.HGT_M.nc
-rw-r--r-- 1 glopara global  3608064000 Feb 18  2014 gmted2010.30sec.flt
-rw-r--r-- 1 glopara global         263 May  5  2014 gmted2010.30sec.flt.ctl
-rw-r--r-- 1 glopara global  1866240000 Aug 14  2017 gmted2010.30sec.int
-rw-r--r-- 1 glopara global         264 May  5  2014 gmted2010.30sec.int.ctl
-rw-r--r-- 1 glopara global  1866240008 Mar 11  2017 gtopo30_gg.fine
-rw-r--r-- 1 glopara global  1866240000 Aug 16  2017 gtopo30_gg.fine.nh
-rw-r--r-- 1 glopara global   933120000 Mar 11  2017 landcover30.fixed
-rw-r--r-- 1 glopara global         240 May  5  2014 makefile
-rw-r--r-- 1 glopara global         313 Aug 14  2017 run.lsf
-rw-r--r-- 1 glopara global   622267212 Mar 11  2017 thirty.second.antarctic.new.bin
-bash-4.2$ ll orog/20220805/
total 64
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C1152
drwxr-sr-x 3 glopara global 4096 Jan  5 15:59 C1152.mx025
lrwxrwxrwx 1 glopara global   11 Jan  5 15:59 C1152.mx025_frac -> C1152.mx025
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C128
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C192
drwxr-sr-x 3 glopara global 4096 Jan  5 16:00 C192.mx050
lrwxrwxrwx 1 glopara global   10 Jan  5 16:00 C192.mx050_frac -> C192.mx050
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C3072
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C3359
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C3445
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C384
drwxr-sr-x 3 glopara global 4096 Jan  5 16:01 C384.mx025
lrwxrwxrwx 1 glopara global   10 Jan  5 16:01 C384.mx025_frac -> C384.mx025
drwxr-sr-x 3 glopara global 4096 Jan 26 15:27 C48
drwxr-sr-x 3 glopara global 4096 Jan  5 16:02 C48.mx500
lrwxrwxrwx 1 glopara global    9 Jan  5 16:02 C48.mx500_frac -> C48.mx500
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C768
drwxr-sr-x 3 glopara global 4096 Jan  5 16:03 C768.mx025
lrwxrwxrwx 1 glopara global   10 Jan  5 16:03 C768.mx025_frac -> C768.mx025
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C96
drwxr-sr-x 3 glopara global 4096 Jan  5 15:48 C96.mx100
lrwxrwxrwx 1 glopara global    9 Jan  5 15:48 C96.mx100_frac -> C96.mx100
lrwxrwxrwx 1 glopara global   34 Jan 18 15:51 GlobalLakeDepth.dat -> ../../raw/orog/GlobalLakeDepth.dat
lrwxrwxrwx 1 glopara global   35 Jan 18 15:51 GlobalLakeStatus.dat -> ../../raw/orog/GlobalLakeStatus.dat
lrwxrwxrwx 1 glopara global   55 Jan 18 15:51 HGT.Beljaars_filtered.lat-lon.30s_res.nc -> ../../raw/orog/HGT.Beljaars_filtered.lat-lon.30s_res.nc
lrwxrwxrwx 1 glopara global   47 Jan 18 15:50 geo_em.d01.lat-lon.2.5m.HGT_M.nc -> ../../raw/orog/geo_em.d01.lat-lon.2.5m.HGT_M.nc
lrwxrwxrwx 1 glopara global   34 Jan 18 15:51 gmted2010.30sec.int -> ../../raw/orog/gmted2010.30sec.int
lrwxrwxrwx 1 glopara global   32 Jan 18 15:51 landcover30.fixed -> ../../raw/orog/landcover30.fixed
lrwxrwxrwx 1 glopara global   46 Jan 18 15:51 thirty.second.antarctic.new.bin -> ../../raw/orog/thirty.second.antarctic.new.bin
```

These raw files aren't used at runtime in the GFS/global-workflow so we don't need to move to the new UFS_UTILS hash because of this cleanup.

**Comment by KateFriedman-NOAA** [comment]
Renamed `C#.mx###_frac` folders in `orog/20220805` to `C#.mx#` and created symlinks for backwards compatibility:
```
-bash-4.2$ pwd
/scratch1/NCEPDEV/global/glopara/fix/orog/20220805
-bash-4.2$ ll | grep mx
drwxr-sr-x 3 glopara global 4096 Jan  5 15:59 C1152.mx025
lrwxrwxrwx 1 glopara global   11 Jan  5 15:59 C1152.mx025_frac -> C1152.mx025
drwxr-sr-x 3 glopara global 4096 Jan  5 16:00 C192.mx050
lrwxrwxrwx 1 glopara global   10 Jan  5 16:00 C192.mx050_frac -> C192.mx050
drwxr-sr-x 3 glopara global 4096 Jan  5 16:01 C384.mx025
lrwxrwxrwx 1 glopara global   10 Jan  5 16:01 C384.mx025_frac -> C384.mx025
drwxr-sr-x 3 glopara global 4096 Jan  5 16:02 C48.mx500
lrwxrwxrwx 1 glopara global    9 Jan  5 16:02 C48.mx500_frac -> C48.mx500
drwxr-sr-x 3 glopara global 4096 Jan  5 16:03 C768.mx025
lrwxrwxrwx 1 glopara global   10 Jan  5 16:03 C768.mx025_frac -> C768.mx025
drwxr-sr-x 3 glopara global 4096 Jan  5 15:48 C96.mx100
lrwxrwxrwx 1 glopara global    9 Jan  5 15:48 C96.mx100_frac -> C96.mx100
```

Within the `C#.mx###` folder, rename oro files and create symlinks with prior names for backwards compatibility:
```
-bash-4.2$ ll *mx*/oro_*
lrwxrwxrwx 1 glopara global 29 Jan  5 15:59 C1152.mx025/oro_C1152.mx025.tile1.nc -> C1152.mx025_oro_data.tile1.nc
lrwxrwxrwx 1 glopara global 29 Jan  5 15:59 C1152.mx025/oro_C1152.mx025.tile2.nc -> C1152.mx025_oro_data.tile2.nc
lrwxrwxrwx 1 glopara global 29 Jan  5 15:59 C1152.mx025/oro_C1152.mx025.tile3.nc -> C1152.mx025_oro_data.tile3.nc
lrwxrwxrwx 1 glopara global 29 Jan  5 15:59 C1152.mx025/oro_C1152.mx025.tile4.nc -> C1152.mx025_oro_data.tile4.nc
lrwxrwxrwx 1 glopara global 29 Jan  5 15:59 C1152.mx025/oro_C1152.mx025.tile5.nc -> C1152.mx025_oro_data.tile5.nc
lrwxrwxrwx 1 glopara global 29 Jan  5 15:59 C1152.mx025/oro_C1152.mx025.tile6.nc -> C1152.mx025_oro_data.tile6.nc
lrwxrwxrwx 1 glopara global 29 Jan  5 15:59 C1152.mx025_frac/oro_C1152.mx025.tile1.nc -> C1152.mx025_oro_data.tile1.nc
lrwxrwxrwx 1 glopara global 29 Jan  5 15:59 C1152.mx025_frac/oro_C1152.mx025.tile2.nc -> C1152.mx025_oro_data.tile2.nc
lrwxrwxrwx 1 glopara global 29 Jan  5 15:59 C1152.mx025_frac/oro_C1152.mx025.tile3.nc -> C1152.mx025_oro_data.tile3.nc
lrwxrwxrwx 1 glopara global 29 Jan  5 15:59 C1152.mx025_frac/oro_C1152.mx025.tile4.nc -> C1152.mx025_oro_data.tile4.nc
lrwxrwxrwx 1 glopara global 29 Jan  5 15:59 C1152.mx025_frac/oro_C1152.mx025.tile5.nc -> C1152.mx025_oro_data.tile5.nc
lrwxrwxrwx 1 glopara global 29 Jan  5 15:59 C1152.mx025_frac/oro_C1152.mx025.tile6.nc -> C1152.mx025_oro_data.tile6.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:00 C192.mx050/oro_C192.mx050.tile1.nc -> C192.mx050_oro_data.tile1.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:00 C192.mx050/oro_C192.mx050.tile2.nc -> C192.mx050_oro_data.tile2.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:00 C192.mx050/oro_C192.mx050.tile3.nc -> C192.mx050_oro_data.tile3.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:00 C192.mx050/oro_C192.mx050.tile4.nc -> C192.mx050_oro_data.tile4.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:00 C192.mx050/oro_C192.mx050.tile5.nc -> C192.mx050_oro_data.tile5.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:00 C192.mx050/oro_C192.mx050.tile6.nc -> C192.mx050_oro_data.tile6.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:00 C192.mx050_frac/oro_C192.mx050.tile1.nc -> C192.mx050_oro_data.tile1.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:00 C192.mx050_frac/oro_C192.mx050.tile2.nc -> C192.mx050_oro_data.tile2.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:00 C192.mx050_frac/oro_C192.mx050.tile3.nc -> C192.mx050_oro_data.tile3.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:00 C192.mx050_frac/oro_C192.mx050.tile4.nc -> C192.mx050_oro_data.tile4.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:00 C192.mx050_frac/oro_C192.mx050.tile5.nc -> C192.mx050_oro_data.tile5.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:00 C192.mx050_frac/oro_C192.mx050.tile6.nc -> C192.mx050_oro_data.tile6.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:01 C384.mx025/oro_C384.mx025.tile1.nc -> C384.mx025_oro_data.tile1.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:01 C384.mx025/oro_C384.mx025.tile2.nc -> C384.mx025_oro_data.tile2.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:01 C384.mx025/oro_C384.mx025.tile3.nc -> C384.mx025_oro_data.tile3.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:01 C384.mx025/oro_C384.mx025.tile4.nc -> C384.mx025_oro_data.tile4.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:01 C384.mx025/oro_C384.mx025.tile5.nc -> C384.mx025_oro_data.tile5.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:01 C384.mx025/oro_C384.mx025.tile6.nc -> C384.mx025_oro_data.tile6.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:01 C384.mx025_frac/oro_C384.mx025.tile1.nc -> C384.mx025_oro_data.tile1.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:01 C384.mx025_frac/oro_C384.mx025.tile2.nc -> C384.mx025_oro_data.tile2.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:01 C384.mx025_frac/oro_C384.mx025.tile3.nc -> C384.mx025_oro_data.tile3.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:01 C384.mx025_frac/oro_C384.mx025.tile4.nc -> C384.mx025_oro_data.tile4.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:01 C384.mx025_frac/oro_C384.mx025.tile5.nc -> C384.mx025_oro_data.tile5.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:01 C384.mx025_frac/oro_C384.mx025.tile6.nc -> C384.mx025_oro_data.tile6.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 16:02 C48.mx500/oro_C48.mx500.tile1.nc -> C48.mx500_oro_data.tile1.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 16:02 C48.mx500/oro_C48.mx500.tile2.nc -> C48.mx500_oro_data.tile2.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 16:02 C48.mx500/oro_C48.mx500.tile3.nc -> C48.mx500_oro_data.tile3.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 16:02 C48.mx500/oro_C48.mx500.tile4.nc -> C48.mx500_oro_data.tile4.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 16:02 C48.mx500/oro_C48.mx500.tile5.nc -> C48.mx500_oro_data.tile5.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 16:02 C48.mx500/oro_C48.mx500.tile6.nc -> C48.mx500_oro_data.tile6.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 16:02 C48.mx500_frac/oro_C48.mx500.tile1.nc -> C48.mx500_oro_data.tile1.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 16:02 C48.mx500_frac/oro_C48.mx500.tile2.nc -> C48.mx500_oro_data.tile2.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 16:02 C48.mx500_frac/oro_C48.mx500.tile3.nc -> C48.mx500_oro_data.tile3.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 16:02 C48.mx500_frac/oro_C48.mx500.tile4.nc -> C48.mx500_oro_data.tile4.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 16:02 C48.mx500_frac/oro_C48.mx500.tile5.nc -> C48.mx500_oro_data.tile5.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 16:02 C48.mx500_frac/oro_C48.mx500.tile6.nc -> C48.mx500_oro_data.tile6.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:03 C768.mx025/oro_C768.mx025.tile1.nc -> C768.mx025_oro_data.tile1.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:03 C768.mx025/oro_C768.mx025.tile2.nc -> C768.mx025_oro_data.tile2.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:03 C768.mx025/oro_C768.mx025.tile3.nc -> C768.mx025_oro_data.tile3.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:03 C768.mx025/oro_C768.mx025.tile4.nc -> C768.mx025_oro_data.tile4.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:03 C768.mx025/oro_C768.mx025.tile5.nc -> C768.mx025_oro_data.tile5.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:03 C768.mx025/oro_C768.mx025.tile6.nc -> C768.mx025_oro_data.tile6.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:03 C768.mx025_frac/oro_C768.mx025.tile1.nc -> C768.mx025_oro_data.tile1.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:03 C768.mx025_frac/oro_C768.mx025.tile2.nc -> C768.mx025_oro_data.tile2.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:03 C768.mx025_frac/oro_C768.mx025.tile3.nc -> C768.mx025_oro_data.tile3.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:03 C768.mx025_frac/oro_C768.mx025.tile4.nc -> C768.mx025_oro_data.tile4.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:03 C768.mx025_frac/oro_C768.mx025.tile5.nc -> C768.mx025_oro_data.tile5.nc
lrwxrwxrwx 1 glopara global 28 Jan  5 16:03 C768.mx025_frac/oro_C768.mx025.tile6.nc -> C768.mx025_oro_data.tile6.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 15:48 C96.mx100/oro_C96.mx100.tile1.nc -> C96.mx100_oro_data.tile1.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 15:48 C96.mx100/oro_C96.mx100.tile2.nc -> C96.mx100_oro_data.tile2.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 15:48 C96.mx100/oro_C96.mx100.tile3.nc -> C96.mx100_oro_data.tile3.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 15:48 C96.mx100/oro_C96.mx100.tile4.nc -> C96.mx100_oro_data.tile4.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 15:48 C96.mx100/oro_C96.mx100.tile5.nc -> C96.mx100_oro_data.tile5.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 15:48 C96.mx100/oro_C96.mx100.tile6.nc -> C96.mx100_oro_data.tile6.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 15:48 C96.mx100_frac/oro_C96.mx100.tile1.nc -> C96.mx100_oro_data.tile1.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 15:48 C96.mx100_frac/oro_C96.mx100.tile2.nc -> C96.mx100_oro_data.tile2.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 15:48 C96.mx100_frac/oro_C96.mx100.tile3.nc -> C96.mx100_oro_data.tile3.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 15:48 C96.mx100_frac/oro_C96.mx100.tile4.nc -> C96.mx100_oro_data.tile4.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 15:48 C96.mx100_frac/oro_C96.mx100.tile5.nc -> C96.mx100_oro_data.tile5.nc
lrwxrwxrwx 1 glopara global 27 Jan  5 15:48 C96.mx100_frac/oro_C96.mx100.tile6.nc -> C96.mx100_oro_data.tile6.nc
```

**Comment by WalterKolczynski-NOAA** [comment]
Completed by #2181
